conda_build/main_build.py

```diff
@@ -308,6 +309,8 @@ def execute(args, parser):
                     (conda_version[lang], version))
 
     if args.skip_existing:
+        if not isdir(config.bldpkgs_dir):
+            makedirs(config.bldpkgs_dir)
```

Comment:
Conda code is a mess, you have to copy things over and over. There are two other places with the same code:
https://github.com/conda/conda-build/blob/master/conda_build/build.py#L273
https://github.com/conda/conda-build/blob/master/conda_build/build.py#L486 (I have no idea if makedirs is necessary there)